opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -82,20 +82,24 @@ message Span {
   // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes
   // is considered invalid.
   //
-  // This field is semantically required. Receiver should generate new
-  // random trace_id if empty or invalid trace_id was received.
-  //
```

Comment:
Sorry, this is not a specification document. I would discard this language and treat spans w/o valid trace IDs and span IDs as invalid spans. Why change this now?

Reply:
Agreed.  By the time span data are represented as OTLP we're out of the SDK and the data are either valid or not.

Reply:
>Why change this now?

I am not changing it. It is already like that, see https://github.com/open-telemetry/opentelemetry-proto/blob/d8729d40f629dba12694b44c4c32c1eab109b00a/opentelemetry/proto/trace/v1/trace.proto#L85

Reply:
https://github.com/open-telemetry/opentelemetry-proto/pull/444 is now merged and we no longer generate random ids.